src/main/pages/che-7/administration-guide/assembly_using-organizations.adoc

```diff
@@ -0,0 +1,224 @@
+---
+title: "Using organizations"
+keywords: organizations, user management, permissions
+tags: [organizations]
+sidebar: che_7_docs
+permalink: che-7/using-organizations/
+redirect_from:
+  - organizations.html
+folder: che-7/administration-guide
+---
+
+:parent-context-of-using-organizations: {context}
+
+:context: using-organizations
+
+[id="organizations-in-eclipse-che"]
+== Organizations in Eclipse Che
+
```

Comment:
in this document, we should highlight the capabilities that are working from within Che. Which ones are still functional on K8S and OpenShift.
cc @skabashnyuk and @sleshchenko

Reply:
@slemeur can you confirm the intention to remove organizations https://github.com/eclipse/che/issues/15304#issuecomment-558880903 ?

Reply:
NO. there is no intention to remove the organizations, which are still features that are important and beneficial in a cloud IDE. Teams and collaboration aspects are big part of the value of Che, why would you remove it?
Instead of thinking about removing it, we need to analyze properly the capabilities that are working and the ones that are not working. 
Based on this analyze, we can adjust the functionalities and guide the user (or admin in this case) to either use the underlying platform to operate the some actions or understand the limited scope of the actions from the dashboard.